### ISSUE TITLE
Convert rarreg.key to rarreg.rar?

### ISSUE THREAD

**Issue by teknixstuff** [bitcookies/winrar-keygen#23]
When you purchase WinRAR, you get a rarreg.rar file which can be opened in winrar to activate it.
How can I convert the rarreg.key that this tool generates to a rarreg.rar? Just compressing it doesn't work.

**Comment by bitcookies** [comment]
`rarreg.rar` is just a compressed file for WinRAR, the `.key` is the actual form of the license file that does the work.

You can drag `rarreg.key` to the WinRAR GUI window or move it to the following path

```
C:\Users\yourname\AppData\Roaming\WinRAR\rarreg.key
```

See https://github.com/bitcookies/winrar-keygen/issues/4#issuecomment-887233951

**Comment by teknixstuff** [comment]
Yes, I know how to use the rarreg.key without rarreg.rar, but making a rarreg.rar would be a nice feature.

**Comment by bitcookies** [comment]
According to the [official winrar video](https://www.youtube.com/watch?v=LZlGRF_6IyU), the `rarreg.key` file needs to be compressed into `rarkey.rar` , not `rarreg.rar`.

Double-click the `rarkey.rar` file to open it and it will be automatically activated.

The winrar website gives [incorrect naming information for the rarreg.rar file](https://www.win-rar.com/faq0.html?&L=1).
